dotcom-rendering/src/components/AffiliateDisclaimer.tsx

```diff
@@ -52,15 +65,23 @@ const AffiliateDisclaimer = () => (
 	</Hide>
 );
 
-const AffiliateDisclaimerInline = () => (
-	<Hide from="leftCol">
+const AffiliateDisclaimerInline = ({ isAmp = false }) =>
+	isAmp ? (
 		<aside
-			css={[disclaimerInlineStyles]}
+			css={[disclaimerInlineStyles, ampStyles]}
```

Comment:
could have `{isAmp ? ampDisclaimerStyles : disclaimerInlineStyles}` rather than the `if` the whole block I think?
Although the `Hide` isn't needed on amp, I don't think it'll break anything?

Reply:
Ahhh of course 🤦‍♂️

Reply:
This sort of inherent difference beween Web & AMP is why [we might want to keep the components distinct](https://github.com/guardian/dotcom-rendering/pull/11164#pullrequestreview-2003052419).